spec/features/visitor_creates_account_spec.rb

```diff
@@ -3,10 +3,22 @@
 feature "Account Creation" do
   let(:subdomain) { generate(:subdomain) }
 
-  scenario "allows a guest to create an account" do
-    sign_up(subdomain)
-    expect(page.current_url).to include(subdomain)
-    expect(Account.count).to eq(1)
+  context "allows a guest to create an account" do
+    scenario "application uses single tenant" do
+      allow(Rails).to receive('ENV["SINGLE_TENANT_MODE"]').and_return('true')
+      allow(Hours).to receive('single_tenant_mode?').and_return true
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.